bigquery/table.go

```diff
@@ -947,6 +1057,18 @@ func (tm *TableMetadataToUpdate) toBQ() (*bq.Table, error) {
 		t.DefaultCollation = optional.ToString(tm.DefaultCollation)
 		forceSend("DefaultCollation")
 	}
+	if tm.PrimaryKey != nil || len(tm.ForeignKeys) > 0 {
```

Comment:
Is special logic needed for clearing the TableConstraints message?  ex: if both PK and FK are empty?  Otherwise, how does a user remove TableConstraints?

Reply:
ah good point, I forgot about allowing users to remove constraints. I just pushed an update to allow that.